text/0000-signed-transaction-payload/0000-signed-transaction-payload.md

```diff
@@ -0,0 +1,825 @@
+    <table>
+        <tr>
+            <th>Name</th>
+            <th>Type</th>
+            <th>Description</th>
+        </tr>
+        <tr>
+            <td>Unlock Type</td>
+            <td>varint</td>
+            <td>
+                Set to <strong>value 0</strong> to denote a <i>Signature Unlock Block</i>.
+            </td>
+        </tr>
+        <tr>
+            <td valign="top">Signature <code>oneOf</code></td>
+            <td colspan="2">
+                <details>
+                    <summary>WOTS Signature</summary>
+                    <table>
+                        <tr>
+                            <th>Name</th>
+                            <th>Type</th>
+                            <th>Description</th>
+                        </tr>
+                        <tr>
+                            <td>Signature Type</td>
+                            <td>varint</td>
+                            <td>
+                                Set to <strong>value 0</strong> to denote a <i>WOTS Signature</i>.
+                            </td>
+                        </tr>
+                        <tr>
+                            <td>Signature</td>
+                            <td>ByteArray</td>
+                            <td>The signature signing the serialized <i>Unsigned Transaction</i>.</td>
+                        </tr>
+                    </table>
+                </details>
+                 <details>
+                    <summary>Ed25519 Signature</summary>
+                    <table>
+                        <tr>
+                            <th>Name</th>
+                            <th>Type</th>
+                            <th>Description</th>
+                        </tr>
+                        <tr>
+                            <td>Signature Type</td>
+                            <td>varint</td>
+                            <td>
+                                Set to <strong>value 1</strong> to denote an <i>Ed25519 Signature</i>.
+                            </td>
+                        </tr>
+                        <tr>
+                            <td>Public key</td>
+                            <td>ByteArray[32]</td>
+                            <td>The public key of the Ed25519 keypair which is used to verify the signature.</td>
+                        </tr>
+                        <tr>
+                            <td>Signature</td>
+                            <td>ByteArray[64]</td>
+                            <td>The signature signing the serialized <i>Unsigned Transaction</i>.</td>
+                        </tr>
+                    </table>
+                </details>
+            </td>
+        </tr>
+    </table>
+</details>
+
+A <i>Signature Unlock Block</i> defines an <i>Unlock Block</i> which holds one or more signatures unlocking one or more inputs. Such block signs the entire <i>Unsigned Transaction</i> part of a <i>Signed Transaction</i> including the optional payload.
```

Comment:
The wording here  is misleading: a single Unlock Block contains only one signature; this signature in turns **may** be used to unlock more than its corresponding one input only if referenced by a proceeding Reference Unlock Block.